docs/api_docs/python/README.md

```diff
@@ -2,7 +2,7 @@
 
 Documentation for the Python API is built with Sphinx under this directory.
 When new versions are released, the documentation is generated and published
-to the [Read the Docs](https://readthedocs.org/projects/iree-python-api/)
+to the [Read the Docs](https://app.readthedocs.org/projects/iree-python-api/)
```

Comment:
I think the project page indeed is https://readthedocs.org/projects/iree-python-api/ (?)

Reply:
https://about.readthedocs.com/blog/2024/11/rollout-of-our-new-dashboard/

> You are currently using our legacy dashboard, which will be retired on March 11th, 2025. You should [switch to our new dashboard](https://app.readthedocs.org/accounts/login/) before then. For more information on this change and what to expect, [read our blog post](https://about.readthedocs.com/blog/2024/11/rollout-of-our-new-dashboard/).

Reply:
Oh nice, thanks for pointing me to this.